gradle.properties

```diff
@@ -1,17 +1,15 @@
 # Done to increase the memory available to gradle.
 org.gradle.jvmargs=-Xmx1G
 
-# Fabric Properties
-	# check these on https://fabricmc.net/use
-	minecraft_version=1.14
-	yarn_mappings=1.14+build.1
-	loader_version=0.4.2+build.132
+# Fabric Properties (https://modmuss50.me/fabric.html)
```

Comment:
I see you changed my gradle

Reply:
why did I click review

Reply:
I'm fine with this

Reply:
I just don't know how to use GitHub pull requests properly

Reply:
How do I resolve this

Reply:
lol